packages/eslint/README.md

```diff
@@ -0,0 +1,35 @@
+# webpack-blocks - TSLint
```

Comment:
Copy 🍝 - needs to be `ESLint` as well 🙂

Reply:
Oh, thanks :) These packages are very similar, so I've decided to copy files from the `tslint` block.

Reply:
Sure, that's how I would have done as well 😉